addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_lights.py

```diff
@@ -55,7 +55,7 @@ def __filter_lights_punctual(blender_lamp, export_settings) -> bool:
 def __gather_color(blender_lamp, export_settings) -> Optional[List[float]]:
     emission_node = __get_cycles_emission_node(blender_lamp)
     if emission_node is not None:
-        return emission_node.inputs["Color"].default_value
+        return list(emission_node.inputs["Color"].default_value)[:3]
```

Comment:
The first issue is that it apparently wasn't a list, so needed to be cast to one, or an exception would be thrown later. And after adding the cast to a list, the output value in the .gltf had 4 components, so I removed one. Using 2.79b, in case that matters.